runelite-client/src/main/java/net/runelite/client/plugins/microbot/bankjs/BanksBankStander/BanksBankStanderScript.java

```diff
@@ -166,27 +166,54 @@ private boolean hasItems() {
     private String fetchItems() {
         if (config.pause()) {
             while (this.isRunning() && config.pause()) {
-                if (!config.pause() || !this.isRunning()) {
-                    break;
-                }
+                if (!config.pause() || !this.isRunning()) { break; }
                 sleep(100, 1000);
             }
         }
-        if (currentStatus != CurrentStatus.FETCH_SUPPLIES) {
-            currentStatus = CurrentStatus.FETCH_SUPPLIES;
-        }
+        if (currentStatus != CurrentStatus.FETCH_SUPPLIES) { currentStatus = CurrentStatus.FETCH_SUPPLIES; }
         sleep(calculateSleepDuration());
         if (!hasItems()) {
             if (!Rs2Bank.isOpen()) {
                 Rs2Bank.openBank();
             }
-            sleep = sleepUntilTrue(Rs2Bank::isOpen, random(67, 97), 18000);
+            sleep = sleepUntilTrue(() -> Rs2Bank.isOpen(), Rs2Random.between(67, 97), 18000);
             sleep(calculateSleepDuration());
-            if (firstItemId != null && secondItemId != null && thirdItemId != null && fourthItemId != null) {
+            if (config.depositAll()) {
+                Rs2Bank.depositAll();
```

Comment:
I think it's better if we return early here

return "";

This avoids adding !config.depositAll to the checks under this code